src/style.js

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components';
+import { Box } from 'grommet';
+
+export const HeaderContainer = styled(Box)`
+  position: relative;
+`;
+
+export const CopyButton = styled(Box)`
+  position: absolute;
+  top: 16px;
+  width: 50px;
+  height: 50px;
+  right: 0;
+  visibility: visible;
+
+  @media only screen and (min-width: 768px) {
```

Comment:
Can you pull the `768px` from the grommet theme file? It's best to avoid hardcoding your respond breakpoint.

Reply:
do you mean pulling it from the theme object? theme.global.breakpoints?

Reply:
The value should be pulled from whatever theme file is being passed to grommet-swagger in the html config. this breakpoint should be consistent with the theme provided.